src/app/organisation/index.njk

```diff
@@ -56,25 +58,21 @@
         </a>
       </div>
 
-      <script>
-        $(document).ready(function () {
-          $("#organisation-search").on("keyup", function () {
-            var value = $(this)
-              .val()
-              .toLowerCase();
-            $("#organisation_table_row tr").filter(function () {
-              $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1)
-            });
-          });
-        });
-      </script>
-
-      <div class="govuk-grid-column-full">
-        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
-        <p class="govuk-body-m">Search for people within the organisation,  {{ cookie.getOrganisationName() }}</p>
-        <input class="govuk-input govuk-input--width-20" id="organisation-search" name="organisation-search" type="text" placeholder="Search" style="margin-bottom: 10px">
-      </div>
       <form id="organisation" action="/organisation" class="form-group" method="post" role="form" >
+        <div class="govuk-grid-column-two-thirds">    
+          <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
+          <p class="govuk-body-m">Search for people within the organisation,  {{ cookie.getOrganisationName() }}</p>
+          <input class="govuk-input govuk-input--width-20" id="organisation-search" name="organisation_search" type="text" value="{{ searchUserName }}" placeholder="Search">
```

Comment:
Bug: If you go and search for a user and then (with the value in the search field) decide to edit or delete a value, it will return straight back to the search page:
* reproduce:
  * Search a value.
  * Click on edit ro delete for a particular user.
  * Return to search page not edit ro delete (see screenshots).
  
  
  Instead, you need handle the routing of the `organisation_search` property in the post.controller.js differently.
  * in other parts of the app, `formaction` [attribtues](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#formaction) have been used, but if there a better solution, please use it.

Reply:
<img width="1440" alt="Screenshot 2024-08-07 at 12 47 47" src="https://github.com/user-attachments/assets/cb19cd69-be18-46c7-b356-2f47aaeebc95">
<img width="1440" alt="Screenshot 2024-08-07 at 12 47 53" src="https://github.com/user-attachments/assets/260c2340-f1f2-41fd-80ce-105e0dcc760f">
<img width="1440" alt="Screenshot 2024-08-07 at 12 48 02" src="https://github.com/user-attachments/assets/4985b340-4923-4e61-99e1-d3f1506fa752">
<img width="1440" alt="Screenshot 2024-08-07 at 12 48 18" src="https://github.com/user-attachments/assets/960b233d-30e7-4ee4-9c30-dcb120707105">

Reply:
<img width="1216" alt="image" src="https://github.com/user-attachments/assets/49efc91f-5a4d-415e-99bb-9183db54996f">


**Edit page**

<img width="1216" alt="image" src="https://github.com/user-attachments/assets/23de0b90-5f0c-431c-9a36-21fb08cf3c38">


**After Edit**

<img width="1216" alt="image" src="https://github.com/user-attachments/assets/1e65a0ba-83d5-456f-88c6-0beeb453841a">